### PR TITLE
Execute MoveAvatars repair step only once

### PR DESCRIPTION
**Pull request by MorrisJobke**
- avoids that this expensive repair step is executed on every update - for a more general way we should discuss this in #2270

cc @rullzer @nickvergessen @LukasReschke @schiessle @blizzz

### CODE REVIEW

File: lib/private/Repair/NC11/MoveAvatars.php [hunk 1]
```diff
@@ -33,32 +34,40 @@ class MoveAvatars implements IRepairStep {
 	private $jobList;
 
 	/** @var SystemConfig */
```
Comment (nickvergessen) [inline lib/private/Repair/NC11/MoveAvatars.php]: IConfig

**Comment by blizzz** [comment]
what @nickvergessen said, otherwise looks good

**Comment by MorrisJobke** [comment]
@nickvergessen @blizzz Done :)

**Comment by nickvergessen** [comment]
So this still runs on the first update, when you install 11, sounds odd...

**Comment by MorrisJobke** [comment]
@nickvergessen Why odd? Repair steps need to be executed ;)

**Comment by nickvergessen** [comment]
But also on an instance that does not need fixing?

**Comment by nickvergessen** [comment]
Well since its getting better with this, fine by me 👍

**Comment by rullzer** [comment]
Lets do this 👍

**Pull request merged**